lua/ulx/modules/sh/cfc_owoify.lua

```diff
@@ -0,0 +1,44 @@
+local CATEGORY_TYPE = "Chat"
+local OWOIFYDICT = { ove = "uv", na = "nya", ne = "nye", ni = "nyi", no = "nyo", nu = "nyu", r = "w", l = "w", O = "owo", U = "uwu", wh = "w" }
```

Comment:
It might be good to turn this into a multiline table, like:
```lua
local OWOIFY_DICT = {
    ove = "uv",
    na = "nya",
}
```
.. and so forth.

It might also be nice to alphabetize it.

Reply:
Sure, give me about 5 minutes